runtime/near-vm-runner/src/runner.rs

```diff
@@ -115,15 +108,14 @@ pub fn run_vm(
             wasm_config,
             fees_config,
             promise_results,
-            profile.clone(),
             current_protocol_version,
             cache,
         ),
         #[cfg(not(feature = "wasmer1_vm"))]
         VMKind::Wasmer1 => panic!("Wasmer1 is not supported, compile with '--features wasmer1_vm'"),
     };
-    if let Some(VMOutcome { burnt_gas, .. }) = &outcome {
-        profile.set_burnt_gas(*burnt_gas)
+    if let Some(outcome) = &outcome {
+        outcome.profile.set_burnt_gas(outcome.burnt_gas)
     }
```

Comment:
One more thing to fix in a follow up -- it doesn't make sense that we track the same amount in two places.

Reply:
Sounds good, profile data doesn't need track burnt gas now